pallets/pallet-web3-names/src/benchmarking.rs

```diff
@@ -161,6 +161,28 @@ benchmarks! {
 			amount: <T as Config>::Deposit::get(),
 		});
 	}
+
+	update_deposit {
+		let deposit_owner: AccountIdOf<T> = account("caller", 0, CALLER_SEED);
+		let owner: Web3NameOwnerOf<T> = account("owner", 0, OWNER_SEED);
+		let web3_name_input: BoundedVec<u8, T::MaxNameLength> = BoundedVec::try_from(
+			generate_web3_name_input(T::MaxNameLength::get().saturated_into())
+		).expect("BoundedVec creation should not fail.");
+		let web3_name_input_clone = web3_name_input.clone();
+		let origin_create = T::OwnerOrigin::generate_origin(deposit_owner.clone(), owner);
+
+		make_free_for_did::<T>(&deposit_owner);
+		Pallet::<T>::claim(origin_create, web3_name_input.clone()).expect("Should register the claimed web3 name.");
+
+		let origin = RawOrigin::Signed(deposit_owner.clone());
+	}: _(origin, web3_name_input_clone)
+	verify {
+		let web3_name = Web3NameOf::<T>::try_from(web3_name_input.to_vec()).unwrap();
+		assert_eq!(Owner::<T>::get(&web3_name).expect("w3n should exists").deposit, Deposit {
```

Comment:
See point in public-credentials benchmarks.

Reply:
good catch [0ba4e65](https://github.com/KILTprotocol/kilt-node/pull/427/commits/0ba4e65c87228e5fbcdbcf00d661c4dfef29fc62)